pkg/kubernetes/manifest.json

```diff
@@ -2,9 +2,17 @@
     "version": 0,
     "discovery": [ "kubernetes" ],
 
+    "dashboard": {
+        "cluster": {
+            "path": "kubernetes/tool",
+            "label": "Dashboard",
+            "order": 0
+        }
```

Comment:
This should probably be "Cluster" and 1.  It's not installed, but I had it linked into ~/.local and thus got a confusing navigation bar.